api/kroxylicious-api/src/main/java/io/kroxylicious/proxy/service/ClusterEndpointProvider.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.service;
+
+import java.util.Optional;
+
+public interface ClusterEndpointProvider {
+
+    /** Address of the cluster's bootstrap address, includes the port.
+     *
+     * @return cluster's bootstrap address in the form host:port
+     */
+    String getClusterBootstrapAddress();
```

Comment:
nit: can we return something like an endpoint record for host/port for the address methods?

Reply:
Yes, I'll do that.

Reply:
Added `HostPort`.  Guava has a utility class, but I decided against it, as it would added a dependency to the api module.